### PR TITLE
Fix incorrect format specifiers (#173)

### PR DESCRIPTION
**Pull request by AMS21**

### CODE REVIEW

File: src/xrCore/Animation/SkeletonMotions.cpp [hunk 1]
```diff
@@ -332,7 +332,7 @@ void motions_container::dump()
         sz += it->second->mem_usage();
         Msg("#%3d: [%3d/%5d Kb] - %s", k, it->second->m_dwReference, it->second->mem_usage() / 1024, it->first.c_str());
     }
-    Msg("--- items: %d, mem usage: %d Kb ", container.size(), sz / 1024);
+    Msg("--- items: %uz, mem usage: %uz Kb ", container.size(), sz / 1024);
```
Comment (Xottab-DUTY) [inline src/xrCore/Animation/SkeletonMotions.cpp]: The correct usage is to specify parameter in-between `%` and `u`, so it should be `%zu`, not `%uz`.
```suggestion
    Msg("--- items: %uz, mem usage: %zu Kb ", container.size(), sz / 1024);
```

Please, fix similar things in other files.

File: src/xrGame/debug_text_tree.h
Comment (Xottab-DUTY) [inline src/xrGame/debug_text_tree.h:1]: I've fixed issues in this file in recent commits.

File: src/xrCore/Animation/SkeletonMotions.cpp (hunk 1 above)
Comment (AMS21) [reply src/xrCore/Animation/SkeletonMotions.cpp]: Oh wow, learned something new :)

**Comment by AMS21** [comment]
Rebased and fixed annotated problems

**Review by Xottab-DUTY — APPROVED** [review]

**Comment by Xottab-DUTY** [comment]
Hurray, merge! Thanks!

**Pull request merged**